EDSSharp/Program.cs

```diff
@@ -66,29 +66,46 @@ static void Main(string[] args)
 
 
                         default:
+                            Program.WriteError("Invalid INFILE extension.");
+                            PrintHelpText();
                             return;
 
                     }
                     if(eds != null)
                     {
                         Export(outfile, outtype);
+                        Console.WriteLine("Successful conversion");
+                    }
+                    else
+                    {
+                        Program.WriteError("Invalid XDD INFILE.");
+                        PrintHelpText();
                     }
                 }
                 else
                 {
+                    Program.WriteError("INFILE or OUTFILE missing.");
                     PrintHelpText();
                 }
             }
-            catch(Exception e)
+            catch(Exception)
             {
-                Console.WriteLine(e.ToString());
+                Program.WriteError("Invalid EDS INFILE.");
+                // Console.WriteLine(e.ToString());
```

Comment:
printing the exception is very useful, keep in mind that not everyone can share the input file or have c# knowledge.